### PR TITLE
Update recommended hardware page to reflect multiple detectors

### PR DESCRIPTION
**Pull request by NickM-27**

### CODE REVIEW

**Comment by NickM-27** [comment]
@NateMeyer Let me know what you think, the J4105 and VPU examples are pulled from your post and the N4020 is from a personal laptop I use for occasional intel testing. Does more information need to be provided for OpenVino side?

File: docs/docs/frigate/hardware.md [hunk 1]
```diff
@@ -33,17 +33,40 @@ My current favorite is the Minisforum GK41 because of the dual NICs that allow y
...
+### OpenVino
+
+The OpenVINO detector type is able to run on x86 hosts that have an Intel CPU, GPU, or VPU Hardware (ex: Intel NCS2). Arm64 hosts are also supported with VPU hardware.
+
+Inference speeds vary greatly depending on the CPU, GPU, or VPU used, some known examples are below:
+
+| Name                | Inference Speed | Notes                                                                 |
+| Intel Celeron J4105 | ~ 25 ms         | Inference speeds on CPU were ~ 150 ms                                 |
+| Intel Celeron N4020 | 50 - 200 ms     | Inference speeds on CPU were ~ 800 ms, greatly depends on other loads |
+| Intel NCS2 VPU      | 60 - 65 ms      | May vary based on host device                                         |
```
Comment (NateMeyer) [inline docs/docs/frigate/hardware.md:59]:
```suggestion
| Intel NCS2 VPU      | 60 - 65 ms      | May vary based on host device                                         |
| Intel i5 1135G7     | 10 - 15 ms      |                                                                                           |
```

Here's the number from my laptop.  I just used the number from the detector inference speed, while running a stock test video of some pedestrians in a street. It's a fairly busy scene, but only one source.  Not sure how to make these numbers comparable or what notes you might want to add to this.

**Review by NateMeyer — COMMENTED** [review]
Looks pretty good. A couple suggestions.

File: docs/docs/frigate/hardware.md (hunk 1 above)
Comment (NickM-27) [reply docs/docs/frigate/hardware.md:59]: That's good because I used the exact same test video for testing the Celeron N4020 so results should line up. I agree it is difficult to standardize, I saw values around 200 ms and then after some time it started dropping and stayed around 30 ms. I think there are already a lot of disclaimers around the values, just helps to give people somewhat of an idea what to expect.

File: docs/docs/frigate/hardware.md [hunk 2]
```diff
@@ -33,17 +33,40 @@ My current favorite is the Minisforum GK41 because of the dual NICs that allow y
...
 The USB version is compatible with the widest variety of hardware and does not require a driver on the host machine. However, it does lack the automatic throttling features of the other versions.
 
 The PCIe and M.2 versions require installation of a driver on the host. Follow the instructions for your version from https://coral.ai
 
 A single Coral can handle many cameras and will be sufficient for the majority of users. You can calculate the maximum performance of your Coral based on the inference speed reported by Frigate. With an inference speed of 10, your Coral will top out at `1000/10=100`, or 100 frames per second. If your detection fps is regularly getting close to that, you should first consider tuning motion masks. If those are already properly configured, a second Coral may be needed.
 
-### What does Frigate use the CPU for and what does it use the Coral for? (ELI5 Version)
+### OpenVino
+
+The OpenVINO detector type is able to run on x86 hosts that have an Intel CPU, GPU, or VPU Hardware (ex: Intel NCS2). Arm64 hosts are also supported with VPU hardware.
```
[the remark this answers is not in the document]
Comment (NickM-27) [reply docs/docs/frigate/hardware.md:52]: Ah interesting, I'll put that down for now

File: docs/docs/frigate/hardware.md [hunk 3]
```diff
@@ -33,17 +33,40 @@ My current favorite is the Minisforum GK41 because of the dual NICs that allow y
...
+Inference speeds vary greatly depending on the CPU, GPU, or VPU used, some known examples are below:
+
+| Name                | Inference Speed | Notes                                                                 |
+| Intel Celeron J4105 | ~ 25 ms         | Inference speeds on CPU were ~ 150 ms                                 |
+| Intel Celeron N4020 | 50 - 200 ms     | Inference speeds on CPU were ~ 800 ms, greatly depends on other loads |
+| Intel NCS2 VPU      | 60 - 65 ms      | May vary based on host device                                         |
+
+### TensorRT
+
+The TensortRT detector is able to run on x86 hosts that have an Nvidia GPU.
```
[the remark this answers is not in the document]
Comment (NickM-27) [reply docs/docs/frigate/hardware.md:64]: Similarly would probably want a note on this page for the OpenVino as well (I believe it is listed on the other docs)

[the remark this answers is not in the document]
Comment (NateMeyer) [reply docs/docs/frigate/hardware.md:64]: Yes, that would be good to mention here as well. Can we also put in a link to the detector doc page sections?

[the remark this answers is not in the document]
Comment (NickM-27) [reply docs/docs/frigate/hardware.md:69]: Yep, I reorganized the docs a bit so there aren't so many `and`s and `or`s

**Review by blakeblackshear — APPROVED** [review]
